dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/context/ContextManager.java

```diff
@@ -210,7 +210,7 @@
     private long iQueryInterval = 900;
 
     /** LDAP read timeout. */
-    private Long iReadTimeout;
+    private Long iReadTimeout = DEFAULT_READ_TIMEOUT;
```

Comment:
No reason to do this. The logic for setting the read timeout on the context takes into consideration that the value is not set.

```suggestion
    private Long iReadTimeout;
```